### DISCUSSION TITLE
Implement sample return?

### DISCUSSION THREAD

**Discussion by smudog** [raceintospace/raceintospace#561] [Ideas]
52 years ago today, while some other activities were going on in lunar space, the plucky space probe Luna 15 was preparing to land in the Sea of Crises to retrieve what might have been the first returned samples of the lunar surface, following on a similar attempt the previous month.  It failed, of course, but it was a interesting last minute diversion.  Luna 16 succeeded in September the following year.  While Apollo 18 and 19 were already cancelled weeks before Luna 16's success, the sample return and rover programs that followed probably weakened the case for follow up deep space missions.

Could robotic sample return be implemented in RIS?  One could imagine maybe a combination of Cricket/LK and Explorer/Sputnik for descent and return stage, respectively.  Weight it with enough prestige to turn a strong opponent victory into a weak one, and useful as a face saving exercise if your capsule or large launcher program implodes at the last minute.

**Comment by peyre** [comment]
I think a sample return mission is a great idea - and seems to me the logical hardware for that would be Surveyor/Luna.  It would give players more reason to do the probe landing and something else to do with the probe.  The points for it as a prestige first could be high, maybe 20 since the probe landing is already 16.

The trouble with adding missions is that each has a number assigned to it and you can't just insert one in the middle.  We could maybe redefine one that's a duplicate (as mentioned in #496), though I have other ideas for the dup I found.  Or we could repurpose a mission type that's hardly ever flown.  Neither option would be trivial to implement, as far as I know.

**Reply by hweimer** [reply]
There are also mission slots for rescue missions, which are not active at the moment (and probably should never be, if I correctly remember your comments). If you try to launch one anyway, you will crash the game because the craft to be rescued is treated as a fourth pad. One also needs to be aware that many of the required steps like recovery/landing currently have animations and stills only for manned capsules.

But somehow I have the feeling that this would not be a terribly popular mission, it literally looks like lunar probe landing woes squared.

**Reply by peyre** [reply]
Yes, the rescue missions are the reason I didn't mention adding on past Soyuz LL - that would add the complication of having to decide what to do with the rescue missions.  I don't think they ever went very far, and I don't think we should go ahead and implement them (whereas I would support implementing Zond), yet still I would hesitate before messing with those rescue entries in the data file and also don't believe skipping over those mission types (from 57 to 62) is a good idea.  

I do think we should seriously consider raising the Max R&D of Surveyor/Luna - I feel that as it is, it fails too often given the time and funds you have to put into it.  It has a lower Max R&D than Ranger/Cosmos yet faces more rolls on its missions.

**Reply by hweimer** [reply]
How were the rescue missions were supposed to work in the first place? You equip a standy craft plus crew on another pad and launch it if the first mission gets stranded somewhere? Like STS-125/STS-400?

**Reply by peyre** [reply]
Rescue missions were a holdover from _Liftoff!_.  They were available only if you had all the hardware ready—you couldn't buy it at the last minute.  They also had a 50% chance of not getting off the ground, and suffered a 5% penalty on every step of the mission.  So they were very iffy, and I'm guessing they don't get a lot of use in the board game.  Also I'm pretty sure they were abandoned before movies were created for them.

**Reply by peyre** [reply]
Ok, having a second look at the _Liftoff!_ manual...it's under Optional & Advanced Rules, p.11.  Sounds complicated to implement in a computer game!  Among other things, the ability to fly a mission with only one crew member is one of those special exceptions that complicates things.

> **Rescues:** When using the advanced failure charts, rescues are often necessary. For a rescue attempt to be made, a capable craft must be already on the launch pad ready to go, or a quick-assembly must be undertaken.
> 
> A quick assembly is the putting together of a mission from equipment that has already been purchased but has not been prepared for launch. In this case, the rescue craft has a 50% chance of not getting off the ground, and there is a 5% penalty on every safety check.
> 
> To perform the rescue, the rescue craft may perform any mission that gets it to the same balloon as the stranded craft on the game map. Make sure that the craft being sent also has room for all the astronauts that need to be rescued (this is the only case where a partrial crew maybe launched, for example a three-person capsule with only one astronaut on board). Once in the vicinity of the stranded astronauts, the rescue craft may attempt a docking, or the astronauts may attempt to go to the rescue vehicle by using an emergency EVA.
> 
> For an emergency EVA to be successful, an EVA safety check must be made against the EVA safety factor divided in half. If the emergency EVA and docking are both unsuccessful, the rescue is a failure, and the rescuing ship must return to earth.
> 
> Another means of rescue exists if a craft is in the same mission step as the stranded craft in the same month (both missions were rushed the same amount). If this is the case, the rescuing craft must roll on the ''Unplanned Orbital Rendevous" chart in Appendix E before attempting a docking or emergency EVA.

**Reply by hweimer** [reply]
Thanks for sharing the text! I agree with you that implementing this would be somewhat complicated, especially since getting stranded is kind of a rare event in RIS (most of my 'nauts tend to die in other ways). Is this different in Liftoff? Requiring a larger capsule for the rescue is also a deal-breaker in many cases.

**Reply by peyre** [reply]
Looking over the advanced failure charts, there's a number of entries that say they will be catastrophic failures unless a rescue is effected.  It seems to happen mostly in cases where a vessel is stranded due to engine failure...so, not all that often, no.  Also, how would the game handle them?  I think it's rare that someone would _schedule_ a rescue mission, and the affected crew doesn't have 6 months' worth of consumables, so would a qualifying failure prompt the user to choose whether to fly a rescue mission immediately?  

Additionally, you would need a direct ascent capsule to rescue an Apollo/Soyuz crew, and nothing could rescue a direct ascent crew.

**Reply by peyre** [reply]
Oh, I never did address your comment that a sample return mission seems like something that wouldn't be very popular.  Just guessing here, but I suspect that no, it wouldn't be too popular with most people, but might be with history hounds.  It might help if a successful sample return would give the player an LM point or something.

Personally I'd be more interested in repurposing a mission type to give Mercury/Vostok another prestige mission so we can remove its (ahistorical) EVA capability (#393).  However, I do like the idea of adding a sample return mission, if we can do so without disrupting the rest of the game (like by messing up the order of missions etc.).
